hubspot-client-bundles/pom.xml

```diff
@@ -58,6 +59,16 @@
         <artifactId>hbase-shaded-netty-tcnative</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.hbase</groupId>
+        <artifactId>hbase-compression-zstd</artifactId>
```

Comment:
This should already be defined in the hbase parent. The only reason I have the others is because I needed to add some exclusions

Reply:
done